tests/unit/test_scraper.py

```
@@ -21,830 +21,339 @@
-        assert isinstance(prices[0], dict)
-        assert 'model' in prices[0]
-        assert 'price' in prices[0]
-
-def test_get_kaitori_prices_retry(mock_config):
-    """買取価格取得のリトライテスト"""
-    scraper = Scraper(mock_config)
-    with patch('requests.Session.get', side_effect=Exception("Connection error")):
-        with pytest.raises(ScraperError) as exc_info:
-            scraper.get_kaitori_prices()
-        assert "買取価格の取得に失敗" in str(exc_info.value)
+def test_parse_response(scraper):
```

Ruff (0.8.2)

Redefinition of unused `test_parse_response` from line 85

(F811)